scalardl-test/verification-config.toml

```diff
@@ -24,7 +24,13 @@
   is_verification = true
   num_accounts = 10
   population_concurrency = 32
+
+[storage_config]
   contact_points = "localhost"
```

Comment:
It looks like we need to change the verification script side for this change?

Reply:
@brfrn169 Thank you for the suggestion!
As you said, yes, it's needed. I made a PR https://github.com/scalar-labs/scalar/pull/551 .

Reply:
Thank you!